roles/generate-docs/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# tasks file for gen-docs
+
+- import_tasks: "create_readme.yml"
```

Comment:
Since this role is very very simple, let's just add the code to the `main.yml` to avoid yet another `import` call here.